cypress/support/commands.js

```diff
@@ -23,3 +23,12 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+    cy.get(`[placeholder="${placeholder}"]`);
```

Comment:
You should return the result of the 'cy.get()' call in the 'findByPlaceholder' command to ensure proper chaining 🌟.